### PR TITLE
Fix CVE-2017-1000487 security alert

### PR DESCRIPTION
**Pull request by xumia**
#### Why I did it
Plexus-utils before 3.0.16 is vulnerable to command injection because it does not correctly process the contents of double quoted strings.

#### How I did it
Upgrade to 3.0.16

#### Which release branch to backport (provide reason below if selected)

- [x] 201811
- [x] 201911
- [x] 202006
- [x] 202012

### CODE REVIEW

**Comment by xumia** [comment]
https://www.debian.org/security/2018/dsa-4146

File: src/thrift/patch/0002-cve-2017-1000487.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- ./thrift-0.11.0/contrib/thrift-maven-plugin/pom.xml	2021-03-29 05:20:22.046039640 +0000
++++ ./thrift-0.11.0/contrib/thrift-maven-plugin/pom_new.xml	2021-03-29 05:48:30.725383841 +0000
```
Comment (qiluo-msft) [inline src/thrift/patch/0002-cve-2017-1000487.patch:2]: Is this a patch from upstream? If not, why did you change filename?

Comment (xumia) [reply src/thrift/patch/0002-cve-2017-1000487.patch:2]: @qiluo-msft , can we simply change ./thrift-0.11.0/contrib/thrift-maven-plugin/pom_new.xml to ./thrift-0.11.0/contrib/thrift-maven-plugin/pom.xml.
I tested src/thrift/patch/0002-cve-2017-1000487.patch, there is no difference, when applying the patch by below command:
patch -p1 < ../patch/0002-cve-2017-1000487.patch

**Review by qiluo-msft — APPROVED** [review]

**Comment by yxieca** [comment]
@xumia this change cannot be cherry-picked into 201811 branch cleanly. Can you create an PR for 201811 branch please?

**Pull request merged**